tilelang/intrinsics/wmma_macro_generator.py

```
@@ -201,13 +204,17 @@ def _warp_ldmatrix_a(A_local_buf, A_shared_buf, ki, thread_binding, rk=0):
                     for local_id in T.vectorized(k_pack * local_size_a):
                         row, col = T.meta_var(reverse_index_map(tx, local_id))
                         l, r = (rk * chunk + ki * (k_pack * micro_size_k), warp_m * warp_row_tiles + i * micro_size_x)
```

Ruff (0.15.9)

[error] Ambiguous variable name: `l`

(E741)